plugins/coverimage.koplugin/main.lua

```diff
@@ -259,7 +272,41 @@ function CoverImage:addToMainMenu(menu_items)
                 end,
                 sub_item_table = {
                     {
-                        text = _("Scale, black background"),
+                        text_func = function()
+                            return T(_("Elastic stretch limit (%1%)"), self.cover_image_stretch_limit )
+                        end,
+                        help_text_func = function()
+                            return T(_("If the aspect ratios of the image and the screen differ less than the selected limit (%1%), the image gets stretched (instead of scaled) to fullscreen."), self.cover_image_stretch_limit )
```

Comment:
`If the image and the screen have a similar aspect ratio (more or less %1%), stretch the image instead of honoring its aspect ratio.`

Reply:
Should we use +/- or the cooler ± ?

Reply:
The cooler one, I had intended to lookup the codepoint and... then I forgot :D.